### PR TITLE
gutenprint: bump to 5.3.4

### PR DESCRIPTION
**Pull request by kenmays**
Tested on Haiku R1B2.
Bump nightly builds to gutenprint9 - ref: https://github.com/haiku/haiku/commit/89f1fd6512c10854013d231a85289797e47c7543

### CODE REVIEW

File: net-print/gutenprint/gutenprint9-5.3.4.recipe [hunk 1]
```diff
@@ -11,23 +11,24 @@ replacing the Gimp-Print 4.2-based plugin supplied with GIMP 1.2, \
...
+ARCHITECTURES="x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
 	gutenprint9$secondaryArchSuffix = $portVersion
 	lib:libgutenprint$secondaryArchSuffix = 9.4.0 compat >= 9
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libintl
```
Comment (extrowerk) [inline net-print/gutenprint/gutenprint9-5.3.4.recipe:31]: Sec. arch?

**Comment by korli** [comment]
Please don't remove gutenprint8.

**Comment by ModeenF** [comment]
Nice to see some work on this.  I was thinking of updating haiku or perhaps If it was better to move the haiku add-on here, so we can update gutenprint without  building new add-on.

**Comment by kenmays** [comment]
@ModeenF - check with @korli , main thing is to reiew and update add-on compatibility with Gutenprint 5.3.4 before Haiku R1B3 release..

File: net-print/gutenprint/gutenprint8-5.3.1.recipe [hunk 2]
```diff
@@ -13,21 +13,22 @@ and beyond."
...
+ARCHITECTURES="x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
 	gutenprint8$secondaryArchSuffix = $portVersion
 	lib:libgutenprint$secondaryArchSuffix = 8.1.0 compat >= 8
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
```
Comment (Begasus) [inline net-print/gutenprint/gutenprint8-5.3.1.recipe:31]: @korli gettext is GPL licensed, if this is not an issue we can merge this?

**Comment by kallisti5** [comment]
@korli why are we maintaining a gutenprint8 and a gutenprint9?  The version spread is tiny.  Some kind of major api change?

People are asking why we're shipping the *really* old version in gutenprint8.  Since gutenprint8 != gutenprint9, the build-packages hardlink tool never updates the package without manually adjusting.

**Comment by korli** [comment]
@kallisti5 gutenprint changes the SONAME in 5.3.4 from 5.3.3, and the Haiku package depends on version 8.

**Pull request merged**